pkgs/tools/bootloaders/limine/default.nix

```diff
@@ -0,0 +1,31 @@
+{ pkgs, lib, fetchurl, llvm } :
+
+pkgs.stdenv.mkDerivation rec {
```

Comment:
The use of `pkgs` is discouraged, you get all packages as parameters anyway.

Reply:
Done.